### DISCUSSION TITLE
Error when saving filament profiles (Data will only saved for the opened session)

### DISCUSSION THREAD

**Discussion by TheRealDrCarbon** [OrcaSlicer/OrcaSlicer#1279] [Dev]
Hi,

I seem to have discovered a bug. I'm just about to calibrate my material on the X1 Carbon and have accordingly already created a new profile. The problem is that after the complete restart of OrcaSlicer the profiles are not available anymore. Not only that. Even an updated printer profile, which has a slightly different start G-code, is not displayed anymore. The only way to see the profile again is to load a saved project who includes this profile. I use Version 1.6.3

But what is the reason for this? This is of course a critical serious error, if I don't remember or write down the checked printed settings they are gone.

I would like to have an export function to export the profiles to a JSON file or similar to save them offline.

Many greetings

TheRealDrCarbon

**Comment by TheRealDrCarbon** [comment]
And it was created as User-Profile, not as project profile! Maybe there is a programming error?

**Comment by TheRealDrCarbon** [comment]
Hi,
i found a workaroung to this issue!

I find out if i select all possible filaments with the select all button! There i can see my profile! And it will be show in the list agian! Also the changed profiles for printer gcode settings is now available, but it is already strainge!

**Comment by TheRealDrCarbon** [comment]
Ah ok

The problem is bigger than I thought. After the next restart the whole selection is gone again. A possible solution is to deselect all but one filament and reselect them all.

I don't know why this is.

**Comment by briandswan** [comment]
I've encountered the problem as well.  I thought it was something I was doing.  Asked around in Beginning and Pro 3D printing group and no one could come up with a way to reuse the profiles.  I'm starting an Excel sheet with all the brands I'm doing calibration for.  Sad I lost the ones I have done up until this point.

Another thing I have noticed that might be linked.  When I edit the profile, It will NOT let me use the profile in the AMS.  I have to go to the filament and manually load the filament into the AMS and then use the new profile.  I haven't tried a two filament print with it because I don't think the first filament would load back in.
